rust/src/settings.rs

```diff
@@ -103,6 +103,50 @@ pub struct PetSettings {
     /// ```
     pub min_update: usize,
 
+    /// The minimum amount of time reserved for processing messages in the `sum`
+    /// and `sum2` phases, in seconds.
+    ///
+    /// Defaults to 0 i.e. `sum` and `sum2` phases end *as soon as* [`min_sum`]
+    /// messages have been processed. Set this higher to allow for the
+    /// possibility of more than [`min_sum`] messages to be processed in the
+    /// `sum` and `sum2` phases.
+    ///
+    /// # Examples
+    ///
+    /// **TOML**
+    /// ```text
+    /// [pet]
+    /// min_sum_t = 5
+    /// ```
+    ///
+    /// **Environment variable**
+    /// ```text
+    /// XAIN_PET__MIN_SUM_T=5
+    /// ```
+    pub min_sum_t: u64,
+
+    /// The minimum amount of time reserved for processing messages in the
+    /// `update` phase, in seconds.
+    ///
+    /// Defaults to 0 i.e. `update` phase ends *as soon as* [`min_update`]
+    /// messages have been processed. Set this higher to allow for the
+    /// possibility of more than [`min_update`] messages to be processed in the
+    /// `update` phase.
+    ///
+    /// # Examples
+    ///
+    /// **TOML**
+    /// ```text
+    /// [pet]
+    /// min_update_t = 10
+    /// ```
+    ///
+    /// **Environment variable**
+    /// ```text
+    /// XAIN_PET__MIN_UPDATE_T=10
```

Comment:
```suggestion
    /// XAYNET_PET__MIN_UPDATE_T=10
```

Reply:
thanks (need to get used to the new name!)